pkg/services/authz/server.go

```diff
@@ -12,8 +13,9 @@ import (
 	"github.com/grafana/grafana/pkg/storage/legacysql"
 )
 
-func RegisterRBACAuthZService(handler grpcserver.Provider, db legacysql.LegacyDatabaseProvider, tracer tracing.Tracer) {
-	server := rbac.NewService(db, legacy.NewLegacySQLStores(db), log.New("authz-grpc-server"), tracer)
+func RegisterRBACAuthZService(handler grpcserver.Provider, db legacysql.LegacyDatabaseProvider,
+	tracer tracing.Tracer, cache remotecache.CacheStorage) {
```

Comment:
I could do that instead
```suggestion
	tracer tracing.Tracer, cache authlib.Cache) {
```